src/Uno.Extensions.Authentication.MSAL/build/Package.targets

```
@@ -36,4 +36,16 @@
 		</ItemGroup>
 		<Message Importance="high" Text ="Removed: @(_WebView2CoreOutputsToExclude)" />
 	</Target>
+
+	<Target Name="_ValidateMsalDependencyWasm"
+			BeforeTargets="Compile"
+			Condition="'$(IsUnoHead)'=='true' and '$(UnoRuntimeIdentifier)'=='Wasm'">
```

See https://github.com/unoplatform/uno/blob/447017c3f6e3c8abcde3316e6d79aa22101366a4/build/nuget/Uno.WinUI.WebAssembly.props#L6.

```suggestion
			Condition="'$(IsUnoHead)'=='true' and '$(UnoRuntimeIdentifier)'=='WebAssembly'">
```